S3Scanner/__main__.py

```diff
@@ -108,7 +109,10 @@ def scan_single_bucket(s3service, anons3service, do_dangerous, bucket_name):
         if s3service.aws_creds_configured and checkAuthUsersPerms:
             s3service.check_perm_write_acl(b)
 
-    print(f"{b.name} | bucket_exists | {b.get_human_readable_permissions()}")
+    return {
```

Comment:
Just return `b` from this method. I think that will be more efficient - right now `as_dict()` is being called each time this method runs even if the user doesn't want JSON output. 

You'll need to also update the calls to `scan_single_bucket` below.

Reply:
@sa7mon , I went ahead and knocked this one out too.